bundle/jsky.app.ot/src/main/resources/resources/conf/UIInfo.xml

```diff
@@ -105,7 +105,7 @@
             type="siteQuality"
             imageKey="component.gif"
             shortDescription="This component describes the conditions under which this observation can be observed."
-            uiClassName="jsky.app.ot.gemini.editor.EdCompSiteQuality"/>
+            uiClassName="jsky.app.ot.gemini.editor.sitequality.EdCompSiteQuality"/>
```

Comment:
I think we need an OCS emotional support group.
Or Pisco Sours with the cake on Fridays. 😆